marimbabot_hardware/src/hardware_control_node.cpp

```diff
@@ -0,0 +1,34 @@
+#include <ros/ros.h>
+#include <controller_manager/controller_manager.h>
+#include "marimbabot_hardware/servo_interface.hpp"
+
+int main(int argc, char **argv) {
+    ros::init(argc, argv, "hardware_control_node");
```

Comment:
Please rethink the name of the package, executable, the ROS node name here and the code filename.
`hardware_control_node` is very unspecific when this is integrated together with 5 other hardware components each running a driver.